test/evp_test.c

```diff
@@ -102,6 +102,53 @@ static int is_mac_disabled(const char *name);
 static int is_cipher_disabled(const char *name);
 static int is_kdf_disabled(const char *name);
 
+static const char *name_dup_algs[] = {
```

Comment:
I would suggest moving this  to somewhere other than evp_test.c.

Reply:
It was a separate test program originally, and @paulidale suggested moving it into `evp_test.c`, which I did. If you don't have an obvious better place in mind, I'm inclined to leave it be.

Reply:
One of the evp_extra tests maybe?

My suggest was intended to be to add this to an existing test in evp_test so it got good coverage.  I didn't say it like that unfortunately.

Reply:
Yeah, one of the evp_extra test sources would be a better place. Or evp_pkey_provided_test.c

Reply:
Care to suggest a specific spot that looks good to you?

Reply:
Just put it after test_evp_pkey_ctx_dup_kdf() in evp_pkey_provided_test.c.

Reply:
ping @vdukhovni Could you please move the test?